torch/linalg/__init__.py

```diff
@@ -140,6 +140,129 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+eigh = _add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
+
+This function computes the eigenvalues and eigenvectors
+of a complex Hermitian (or real symmetric) matrix :attr:`input` or batches of such matrices
```

Comment:
":attr:\`input\` or batches of" -> ":attr:\`input\`, or a batch of"

Reply:
Thanks! I only changed "eigenvectors :math:`V` is such that" -> "eigenvectors :math:`V` decompose the :attr:`input` such that"